packages/desktop-client/src/components/accounts/AccountSyncCheck.js

```diff
@@ -42,7 +42,13 @@ function getErrorMessage(type, code) {
     default:
   }
 
-  return 'An internal error occurred. Try to login again, or contact help@actualbudget.com for support.';
+  return (
+    <>
+      An internal error occurred. Try to login again, or get{' '}
+      <a href="https://actualbudget.github.io/docs/Contact/">in touch</a> for
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏open in a new tab
```suggestion
      <a href="https://actualbudget.github.io/docs/Contact/" target="_blank">in touch</a> for
```